build/docker/gaia-taskenv/Dockerfile

```diff
@@ -1,14 +1,7 @@
-from taskcluster/tester:0.3.6
+from taskcluster/tester:0.4.4
 maintainer Aus Lacroix <aus@mozilla.com>
 
-# install our custom git plugin
-copy git/target/doc/git-ci-checkout-pr.1 /usr/local/man/man1/git-ci-checkout-pr.1
```

Comment:
Are these no longer used?  I assume they aren't since they're being removed here, but why aren't they used anymore and what is taking their place? Just curious.